src/helics/common/AsioServiceManagerFwd.hpp

```diff
@@ -5,33 +5,10 @@ All rights reserved. See LICENSE file and DISCLAIMER for more details.
 */
 #pragma once
 
-#include <boost/version.hpp>
-
 class AsioServiceManager;
 
-#if (BOOST_VERSION / 100 >= 1066)
-namespace boost
-{
 namespace asio
 {
 class io_context;
 using io_service = io_context;
```

Comment:
Can we get rid of the alias now?

Reply:
Actually can we get rid of this entire file?

Reply:
It should be safe to get rid of the alias, maybe we can get rid of the file -- I think there's at least one file that includes it for the forward declaration.

Reply:
I think the forward declare might be small enough that it may make sense to just do that where it is needed.